examples/web/nextjs-chat/src/app/api/momento/token/config.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  AllDataReadWrite,
+  ExpiresIn,
+  TopicRole, 
+  CacheRole, 
+  TokenScope, 
+  AllTopics, 
+  AllCaches
+} from "@gomomento/sdk";
+
+/**
+ * First, set the scope of permissions for your tokens. 
+ * 
+ * AllDataReadWrite provides read and write permissions to all of your caches:
+ *    export const tokenPermissions: TokenScope =  AllDataReadWrite;
+ * 
+ * You may also provide a bespoke list of permissions for each cache and topic that you have:
+ *    export const tokenPermissions: TokenScope =  {
+ *      permissions: [
+ *        {
+ *          role: CacheRole.ReadWrite | CacheRole.ReadOnly, 
+ *          cache: AllCaches | "your-cache-name"
+ *        },
+ *        {
+ *          role: TopicRole.PublishSubscribe | TopicRole.SubscribeOnly, 
+ *          cache: AllCaches | "your-cache-name",
+ *          topic: AllTopics | "your-topic-name"
+ *        }
+ *      ]
+ *    };
+ * 
+ * More information here: https://docs.momentohq.com/develop/api-reference/auth-tokens#tokenscope-objects
+ */
+export const tokenPermissions: TokenScope = {
+  permissions: [
+    {
+      role: CacheRole.ReadWrite,
```

Comment:
This would actually be cool to be part of the api, maybe query params or something. The user could select a cache and a topic, and then we can make a request to get a scoped token for that specific cache and topic

Reply:
Would that be similar to this? 
https://github.com/momentohq/client-sdk-javascript/blob/2f6541f169644b4025f262b8a489fb71762ca015/packages/core/src/auth/tokens/token-scopes.ts#L4

Would there need to be a way to stack multiple permissions though?

Reply:
Well I was thinking for the demo would be cool for it to just scope it down as low as possible. But we could also just accept a post body of all the permissions that the user wants the token to have

Reply:
I think the point of Anita's config file is to show app owners/maintainers how to restrict the scope (at app deploy time) so that the browser can't get tokens that have permissions to unexpected/unwanted caches. If we make it super dynamic then it's not really that different from just giving out an AllDataReadWrite token.

We can discuss this more for follow-up PRs, if there is something dynamic that is cool and worth demo'ing I'm open to talking about it! But for now this is more about showing how to make it secure.

Reply:
I think the issue that I see here is that now the demo only works if a user has a cause `default-cache` already created

Reply:
yeah I agree with that Matt, see my other comment and see if that addresses your concern

Reply:
tbh i wonder if we should just take the cache chooser out of the app. doesn't have to be in this PR but just thinking.